src/bereikbaarheid/resources/lastbeperking_resource.py

```diff
@@ -23,7 +23,7 @@ def before_import_row(self, row, row_number=None, **kwargs):
         row["lastbeperking_in_kg"] = convert_str(row["lastbeperking_in_kg"], "float")
 
     def skip_row(self, instance, original, row, import_validation_errors=None):
-        if row["lastbeperking_in_kg"] == None:
+        if row["lastbeperking_in_kg"] == None  or not row["lastbeperking_in_kg"]:
```

Comment:
met `not row["lastbeperking_in_kg"]` test je ook of het `None` is. Je kan die test voor `None` dus weghalen. 

Iets uitgebreider. Bij `not var` test je of `var` None, False, [], een lege string, 0 etc is. Dat is dus veel breder dan `var == None`.  De vraag is hier dus: wat wil je eigenlijk testen?